app/upgrades/mainnet/v12/upgrades.go

```diff
@@ -104,6 +118,116 @@ func InitializeStates(
 	auctionKeeper.SetParams(ctx, auctionsV2types.Params{})
 	auctionKeeper.SetAuctionID(ctx, 0)
 	auctionKeeper.SetUserBidID(ctx, 0)
+
+
+	////// refund CMST to vault owner////////
+
+	type refundStruct struct {
```

Comment:
can you move this code from this refund.go or somewhere ?

Reply:
fixed